CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 ### Added
 
 ### Changed
+- `Waiter.wait_until_true` can be set `sleep_duration`
```

Comment:
`Waiter.wait_until_true` can now set an optional `sleep_duration`

Reply:
Sorry to incorrect sentence. I fixed that https://github.com/site-prism/site_prism/pull/105/commits/e954e399351249e73f3e61ad94254d8fb2dd83c1